### ISSUE TITLE
Track forward via bluetooth

### ISSUE THREAD

**Issue by markusd112** [PaulWoitaschek/Voice#866]
Please include the following details in your bug report:

App-Version: 4.3.2
Phone Name: Motorola Moto G6plus
Android Version: 9

I am using Voice in my car (Ford with SYNC3 multimedia system) to play audiobooks. The smartphone is connected via bluetooth to the SYNC3. On the steering wheel are buttons to control the playback. But when I push the track forward/backward buttons, Voice doesn't go to the next / previous track, but moves fast forward/backward several seconds.

Please change the behaviour, so you can go forward/backward one track when pushing these buttons. Thanks.
![IMG_20190411_175523454_HDR](https://user-images.githubusercontent.com/14372683/56013925-669ec400-5cf3-11e9-94e4-8d6d5b8b9e4b.jpg)

**Comment by test2a** [comment]
Why would you want to be able to swit h "tracks" or in audiobooks,  chapters? Arent audiobooks supposed to be listened linearly? A to z?
I suppose the current implementation is fine. Only go back or forward a few seconds if you missed something.

Are you using the app as a music player instead of a dedicated audiobook player?

**Comment by markusd112** [comment]
No, I am using Voice to hear audiobooks, not music ;-). My audiobooks are organized in tracks. I am jumping forward/backward one track (chapter) in different situations, e.g. when I am listening and I want to hear the last chapter again from the beginning (e.g. when I fall into sleep), or I want to jump to a specific chapter. Another use case is when I listen my newspaper audiobook (e.g. DIE ZEIT Audio) where each news article is stored into a single track and I want to go forward to next article (e.g. the article isn't interesting me)

**Comment by test2a** [comment]
Ah. In that case you should suggest an option to change the action button has in the app. 

A simple switch to select between switch tracks and chapters in the settings.

**Comment by markusd112** [comment]
Yes, that would be a feasible way to give to possibility to change the behaviour of the button. Or maybe the app moves forward/backward one track when I double click the button and moves forward/backward in the current track when I press it once or for long time... then you are able to control both with one button. I think that would be the best way....

**Comment by Pentaphon** [comment]
> Are you using the app as a music player instead of a dedicated audiobook player?

That's a good enough possibility to add such a basic feature. Sometimes, people use audiobook players to listen to very long audio recordings without losing their place, like recorded lectures, radio broadcasts, etc.

**Comment by 0rAX0** [comment]
Maybe add a long press action to change the chapter?

**Comment by markusd112** [comment]
Yes, this would be a good solution too... And with this you have both possibilities: fast forward/backward and next/prev. track without changing a setting in the settings dialogue.

**Comment by jtagcat** [comment]
Related: #842
